src/math/abs.jl

```diff
@@ -13,9 +13,9 @@ abs(123.1) # returns 123.1
 1000 == abs_val(1000) #returns true
 ```
 
-Contributed By:- [Ashwani Rathee](https://github.com/ashwani-rathee)
+Contributed By:- [Ashwani Rathee](https://github.com/ashwani-rathee) and [Rratic](https://github.com/Rratic)
 """
-function abs_val(num)
+function abs_val(num::N)where N<:Number
```

Comment:
I guess Number can be changed to Real becuase comparing Complex does not make sense

Reply:
Unless there is a conflict with another `abs_val` method, there is no need to provide types